tests/Unit/Evolution/Particles/MonteCarlo/Test_CommunicationTags.cpp

```diff
@@ -201,6 +203,39 @@ void test_send_receive_actions() {
       get<0>(logical_coordinates(subcell_mesh)) * 2.0);
   std::vector<Particles::MonteCarlo::Packet> packets_on_element{};
 
+  // Logical coordinates
```

Comment:
are you trying to compute the logical coordinates on the subcell mesh?  If so you just call `logical_coordinates(subcell_mesh)`

Reply:
Yeah... I used that function in every line around that section, yet apparently decided to recompute the coordinates by hand...